src/ansible_sign/cli.py

```diff
@@ -357,6 +729,239 @@ def gpg_sign(self):
         self.logger.debug(f"GPG Details: {result.extra_information}")
         return retcode
 
+    def sigstore_sign(self):
```

Comment:
These functions kinda make me wonder if we shouldn't move them out to sign/verify specific modules rather than have it colocated with the cli itself?

Reply:
That would make sense, but in this case should we also move the corresponding GPG commands to stay consistent?